app/views/users/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if @current_user %>
+
+  <h4> Created events</h4>
+
+  <%= render 'created' %>
+
+  <h4>Future events:</h4>
+
+  <%= render 'future' %>
+
+  <h4> Previous events:</h4>
+
+  <%= render 'past' %>
+
+<% end %>
```

Comment:
- Move the if/else statement to a helper method that you call here and render the markup within the helper method using a partial